### PR TITLE
Fix paths for refactored prepocnobs task

### PR DESCRIPTION
**Pull request by AndrewEichmann-NOAA**
# Description

Cycled testing of the refactoring of prepocnobs revealed more issues with paths, which this PR should fix.

Refs https://github.com/NOAA-EMC/GDASApp/issues/1047

# Type of change

- Bug fix (fixes something broken)

# Change characteristics
- Is this a breaking change (a change in existing functionality)? NO
- Does this change require a documentation update? NO

# How has this been tested?

Cycled test on Hera with GDASApp branch `feature/refactor-prepobs`

# Checklist
- [x] Any dependent changes have been merged and published
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] New and existing tests pass with my changes
- [x] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

**Comment by AndrewEichmann-NOAA** [comment]
@guillaumevernieres FYI

File: jobs/JGLOBAL_PREP_OCEAN_OBS [hunk 1]
```diff
@@ -14,13 +14,13 @@ YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COMOUT_OBS:COM_OBS_TMPL
...
-# Add prep_marine_obs.py to PYTHONPATH
-export PYTHONPATH=${HOMEgfs}/sorc/gdas.cd/ush:${PYTHONPATH}
+# Add prep_ocean_obs.py to PYTHONPATH
+export PYTHONPATH=${HOMEgfs}/ush:${PYTHONPATH}
 
 ###############################################################
 # Run relevant script
 
-EXSCRIPT=${GDASPREPOCNOBSPY:-${USHgfs}/exglobal_prep_ocean_obs.py}
+EXSCRIPT=${GDASPREPOCNOBSPY:-${HOMEgfs}/scripts/exglobal_prep_ocean_obs.py}
```
Comment (aerorahul) [inline jobs/JGLOBAL_PREP_OCEAN_OBS]:
```suggestion
EXSCRIPT=${GDASPREPOCNOBSPY:-${SCRgfs}/exglobal_prep_ocean_obs.py}
```

File: jobs/JGLOBAL_PREP_OCEAN_OBS [hunk 2]
```diff
@@ -14,13 +14,13 @@ YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COMOUT_OBS:COM_OBS_TMPL
 # Begin JOB SPECIFIC work
 ##############################################
 
-# Add prep_marine_obs.py to PYTHONPATH
-export PYTHONPATH=${HOMEgfs}/sorc/gdas.cd/ush:${PYTHONPATH}
+# Add prep_ocean_obs.py to PYTHONPATH
+export PYTHONPATH=${HOMEgfs}/ush:${PYTHONPATH}
```
Comment (aerorahul) [inline jobs/JGLOBAL_PREP_OCEAN_OBS]:
```suggestion
```
Can we move this to the rocoto job?
The j-job is not the place to define/update/add `PYTHONPATH`

**Review by aerorahul — CHANGES_REQUESTED** [review]
some minor comments.

File: jobs/JGLOBAL_PREP_OCEAN_OBS (hunk 2 above)
Comment (AndrewEichmann-NOAA) [reply jobs/JGLOBAL_PREP_OCEAN_OBS]: done, and retested

File: jobs/rocoto/prepoceanobs.sh [hunk 3]
```diff
@@ -12,6 +12,11 @@ status=$?
 export job="prepoceanobs"
 export jobid="${job}.$$"
 
+###############################################################
+# setup python path for class defs and utils
+
+export PYTHONPATH=${HOMEgfs}/ush:${PYTHONPATH}
```
Comment (aerorahul) [inline jobs/rocoto/prepoceanobs.sh]:
```suggestion
PYTHONPATH="${PYTHONPATH:+${PYTHONPATH}:}${HOMEgfs}/ush"
```
If `PYTHONPATH` does not exist in the environment, the path will end with `:`

**Review by aerorahul — APPROVED** [review]

**Pull request merged**